bench/bench.py

```diff
@@ -283,15 +282,12 @@ def decomm():
                            help='how many nsqd instances to launch')
     tornado.options.define('worker_count', type=int, default=3,
                            help='how many worker instances to launch')
-    tornado.options.define('access_key', type=str, default='',
-                           help='AWS account access key')
-    tornado.options.define('secret_key', type=str, default='',
-                           help='AWS account secret key')
-    tornado.options.define('ami', type=str, default='ami-48fd2120',
+    # ubuntu 18.04 HVM instance store us-east-1
+    tornado.options.define('ami', type=str, default='ami-0938f2289b3fa3f5b',
                            help='AMI ID')
     tornado.options.define('ssh_key_name', type=str, default='default',
                            help='SSH key name')
-    tornado.options.define('instance_type', type=str, default='c3.2xlarge',
+    tornado.options.define('instance_type', type=str, default='c3.xlarge',
```

Comment:
may I suggest `c5.large`

(which would also require changing the AMI to an HVM-EBS variant, I think ami-0ac80df6eff0e70b5 would do it, and maybe instance-create would then need a root-ebs-size parameter)

... but this could be updated in a separate PR, I could try that after this is merged

Reply:
`c5.large` is VPC-only, so until we rewrite it to setup an entire VPC (not likely), this is Good Enough ™️ 

I didn't mean to change this from the original default, I'll update it.